package.json

```diff
@@ -19,7 +19,8 @@
     "predeploy": "./predeploy.sh",
     "local-deploy": "npm run generate && npm run build && npm run predeploy && npm run serve",
     "prod-deploy": "firebase deploy -P prod",
-    "qa-deploy": "firebase deploy -P qa"
+    "qa-deploy": "firebase deploy -P qa",
+    "prepare": "husky install"
```

Comment:
what does `prepare` do?

Reply:
Enables hooks by default. Prepare is a lifecycle of `npm`. It won't do the trick for `yarn2`
See Step 3 of Husky's [installation guide](https://typicode.github.io/husky/#/?id=create-a-hook) 

3. To automatically have Git hooks enabled after install, edit package.json
`npm set-script prepare "husky install"`